src/js/controller/settings/exportimage/ZipExportController.js

```diff
@@ -71,4 +73,59 @@
   ns.ZipExportController.prototype.getPiskelName_ = function () {
     return this.piskelController.getPiskel().getDescriptor().name;
   };
+
+  ns.ZipExportController.prototype.createZipExportDesc = function (i18n) {
+    var templateData = {
+      divText: i18n.zipExportSettingSectionDesc(),
+    };
+    var templateId = 'zip-export-desc-template';
+    return pskl.utils.Template.fillInTemplate(templateId, templateData);
+  };
+
+  ns.ZipExportController.prototype.createZipExportPrefix = function (i18n) {
+    var templateData = {
+      labelText: i18n.zipExportSettingSectionPrefix(),
+      placeholderText: i18n.zipExportSettingSectionPNGFilePrefix()
+    };
+    var templateId = 'zip-export-prefix-template';
+    return pskl.utils.Template.fillInTemplate(templateId, templateData);
+  };
+
+  ns.ZipExportController.prototype.createZipExportLayers = function (i18n) {
+    var templateData = {
+      labelText: i18n.zipExportSettingSectionSplitByLayers()
+    };
+    var templateId = 'zip-export-layers-template';
+    return pskl.utils.Template.fillInTemplate(templateId, templateData);
+  };
+
+  ns.ZipExportController.prototype.createZipExportDownloadButton = function (i18n) {
+    var templateData = {
+      buttonText: i18n.zipExportSettingSectionDownloadZip()
+    };
+    var templateId = 'zip-export-download-button-template';
+    return pskl.utils.Template.fillInTemplate(templateId, templateData);
+  };
+
+  /**
+  * @private
+  */
+  ns.ZipExportController.prototype.createZipExportDom_ = function (i18n) {
+    var html = '';
+    var zipExportDescHtml = this.createZipExportDesc(i18n);
+    html += zipExportDescHtml;
+    $('#export-info').html(html);
+
+    html = '';
```

Comment:
Minor nitpick. I don't like the whole resetting `html = ''`. I'd rather see the export-info be 1-liner or have `var exportHtml`. Doing the reset like that for an accumulator isn't too bad here, but its usually a bad pattern to follow because it makes the code harder to read. Imagine if this function was 5 times as long and did multiple resets of the `html` variable. It would get tricking remembering the current state of `html` while you are reading the code and is likely to result in bugs.
```suggestion
    $('#export-info').html(this.createZipExportDesc(i18n));

    var html = '';
```